src/hooks/queries/useSafeTx.ts

```diff
@@ -0,0 +1,22 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { fetchSafeTransactionInfo } from '@shapeshiftoss/swapper/dist/safe-utils'
+import { type SafeTxInfo } from '@shapeshiftoss/swapper/dist/safe-utils/types'
```

Comment:
ocd nitpick nested `type`, and also feels like we should be exporting these accordingly from swapper so we don't have to fetch from dist?

Reply:
b6e727b1769ad9e273c73948dd1346a6ab9efb32 - taking a look at exporting these

Reply:
ee49b27ff7

Reply:
And reverted `utils` `src` consumptions in 526df0018f as it rugged vitest: https://github.com/shapeshift/web/actions/runs/10495987074/job/29075843601?pr=7506